test/selenium/signInPage.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
Nits:
* This file should under `test/selenium/lib`. 
* The file name should lower case only (otherwise platforms like Windows or (some) Mac OS will have trouble if another is called `signinpage.js`)